common/src/main/java/com/microsoft/identity/common/internal/ui/embeddedwebview/OAuth2WebViewClient.java

```diff
@@ -47,7 +48,7 @@
 
     private final IChallengeCompletionCallback mCompletionCallback;
     private final AuthorizationRequest mRequest;
-    private final Context mContext;
+    private final Activity mActivity;

```

Comment:
This field appears to be used only in the creation of the `NtlmChallengeHandler` -- I suspect you can use [getContext()](https://developer.android.com/reference/android/view/View.html#getContext()) and avoid this entirely

Reply:
Actually, activity is required to construct the webview. And it is also used in the client cert request and other requests, like open link in browser and install brokers.